docs/concepts/materializing-features.md

```diff
@@ -6,11 +6,11 @@ parent: Feathr Concepts
 
 # Feature Generation and Materialization
```

Comment:
Title could also be updated to echo `Feature materialization (also known as feature generation) `

Reply:
fixed